packages/dds/sequence/src/sequence.ts

```diff
@@ -192,36 +192,16 @@ export abstract class SharedSegmentSequence<T extends ISegment>
         super.on("newListener", (event) => {
```

Comment:
on("newListener" should be removed, and just keep the inside. you don't want to setup on the callback on every listener, just do it once here in the constructor